celery/app/base.py

```diff
@@ -816,6 +817,33 @@ def send_task(self, name, args=None, kwargs=None, countdown=None,
         ignore_result = options.pop('ignore_result', False)
         options = router.route(
             options, route_name or name, args, kwargs, task_type)
+
+        is_native_delayed_delivery = conf.broker_native_delayed_delivery and (
+            conf.broker_url.startswith('amqp://') or conf.broker_url.startswith('py-amqp://'))
```

Comment:
Please fix like in https://github.com/celery/celery/pull/9294

It was originally my code I know. This PR improves upon it.